DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/UserController.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Data.Entity;
+using GitHubExtension.Security.DAL.Infrastructure;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using Microsoft.AspNet.Identity;
+using GitHubExtension.Security.DAL.Interfaces;
+using GitHubExtension.Security.StorageModels.Identity;
+using System.Security.Claims;
+
+
+namespace GitHubExtension.Security.WebApi.Library.Controllers
+{
+
+    public class UserController:BaseApiController
```

Comment:
It may be better. I've created UserController as a controller connected to user preferences.